test/ae_mdw_web/controllers/tx_controller_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule AeMdwWeb.TxControllerTest do
```

Comment:
In the browser works. For CI purposes this can be an integration or `Util.last_gen()` mocked.